config/resources/domain.json

```diff
@@ -584,34 +585,50 @@
     "recognitions": {
       "name": "recognition",
       "class": "fv:Erkenning",
+      "attributes": {
+        "date-document": {
+          "predicate": "eli:dateDocument",
+          "type": "date"
+        },
+        "legal-resource": {
+          "predicate": "eli:hasLegalResource",
+          "type": "string"
+        }
+      },
       "relationships": {
-        "geldigheidsperiode": {
+        "validity-period": {
           "predicate": "fv:geldigheidsperiode",
           "target": "period",
           "cardinality": "one"
         },
-        "toegekendDoor": {
+        "awarded-by": {
           "predicate": "fv:toegekendDoor",
           "target": "administrative-unit",
           "cardinality": "one"
+        },
+        "association": {
+          "target": "association",
+          "predicate": "fv:heeftErkenning",
+          "cardinality": "one",
+          "inverse": true
         }
       },
-      "new-resource-base": "http://awesome-poc.com/target/"
+      "new-resource-base": "https://data.vlaanderen.be/ns/"
```

Comment:
Best to use `http://data.lblod.info/id/` as a base. i.e.:
`http://data.lblod.info/id/erkenningen/`